src/main/webapp/resources/js/components/sequence-files/PairedFileRenderer.jsx

```diff
@@ -15,6 +15,8 @@ import { useSelector } from "react-redux";
  * @function get file processing state function
  * @param qcEntryTranslationKeys Translation keys for qc entries
  * @param displayConcatenationCheckbox Whether to display checkbox or not
+ * @function set default sequencing object for sample
```

Comment:
These should be `@param {function}` not `@function` . See [@function](https://jsdoc.app/tags-function.html)

Reply:
Updated in a23c555bf4b038f2ee0e43742e27ff837b87eb78